core/src/Kokkos_Macros.hpp

```diff
@@ -489,6 +492,9 @@
 #define KOKKOS_ENABLE_DEFAULT_DEVICE_TYPE_HIP
 #elif defined(KOKKOS_ENABLE_SYCL)
 #define KOKKOS_ENABLE_DEFAULT_DEVICE_TYPE_SYCL
+// FIXME_OPENACC: will be enabled after fixing bugs
+//#elif defined(KOKKOS_ENABLE_OPENACC)
+//#define KOKKOS_ENABLE_DEFAULT_DEVICE_TYPE_OPENACC
```

Comment:
Setting the default device type to the OpenACC device is temporarily disabled, since there are many tests running on the default device, which will fail if using features not implemented in the OpenACC backend.
This default device type setting for the OpenACC backend will be re-enabled after all the basic features are implemented.

Reply:
We absolutely need it for View accesses.  The incremental only pass because `Kokkos::abort` does nothing on the device but the standard output is cluttered with memory access violation error messages.

Reply:
I created another local branch, which enables the default OpenACC execution space and disable tests evaluating not-implemented OpenACC features. the new branch executes the incremental tests correctly. I will submit the new PR after the this PR (#5335) is merged.

Reply:
I would do it the other way around.  They are many changes on this PR that are controversial.

Reply:
What you mean is to submit a new PR fixing on the default OpenACC execution space before this PR (#5335)?

Reply:
Yes

Reply:
Submitted a new PR (#5360) for the default OpenACC execution space.

Reply:
PR #5360 is merged into the develop branch and also merged into this PR as a part of rebasing.